components/overview/RedeemUnbonded.js

```diff
@@ -236,8 +250,23 @@ const RedeemUnbonded = withSlideIn(
 									</div>
 									<div className="w-full flex-center">
 										<button
-											className="rounded-full font-medium px-12 py-3 bg-teal-500 mt-40 mb-40 text-white"
+											className={`rounded-full font-medium px-12 py-3 ${
+												controllerAccount &&
+												controllerBalances &&
+												transactionFee +
+													apiInstance?.consts.balances.existentialDeposit.toNumber() >
```

Comment:
change this to transactionFee + (apiInstance?.consts.balances.existentialDeposit.toNumber() / 2)